packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -123,6 +124,33 @@ describe('Facet Test Suite', () => {
       Expect.displaySearchInput(true);
     });
 
+    describe('with fewer values than maximum', () => {
+      const expectedNumberOfValues = defaultNumberOfValues - 1;
+      function setupWithSlicedValues() {
+        mockSliceFacetValues(defaultField, expectedNumberOfValues);
+        cy.visit(pageUrl);
+        configure({
+          field: defaultField,
+          label: defaultLabel,
+          numberOfValues: defaultNumberOfValues,
+        });
+      }
+
+      describe('verify rendering', () => {
+        before(setupWithSlicedValues);
+
+        Expect.displayPlaceholder(false);
+        Expect.labelContains(defaultLabel);
+        Expect.displayValues(true);
+        Expect.numberOfSelectedCheckboxValues(0);
+        Expect.numberOfIdleCheckboxValues(expectedNumberOfValues);
+        Expect.displayClearButton(false);
+        Expect.displayShowMoreButton(true);
```

Comment:
If the facet contains less values than requested, we shouldn't display the "Show more" button, because the facet can't show more values anyways.

Reply:
good point!